java/build/bazel/tests/integration/WorkspaceDriver.java

```diff
@@ -269,14 +302,24 @@ public BazelCommand withEnvironmentVariable(String key, String value) {
     public Command build() {
         String bazelRcPath = bazelrcFile.map(p -> workspace.resolve(p).toString()).orElse("/dev/null");
 
+        Stream<String> experimentalDistdir;
```

Comment:
isn't this a bit of an overkill?
not writing a lot of java these days but isn't a simple string preferable? Maybe an Optional<String> with a getOrElse of empty string?
Also can we move this also to a private method?

Reply:
This is completely an overkill, but I did not want to use a mutable list, which would be clearly indicated here, because I know I am dealing with Scala people here :)  getOrElse of empty string won't work, because you'll end up with an argument of length 0 passed to Bazel, it won't be happy.  I fear that anything else than mutability is an overkill here, but that's my opinion.

Ok for the private method.